package.json

```diff
@@ -30,7 +30,8 @@
 		"Amish Shah <amishshah.2k@gmail.com>",
 		"Vlad Frangu <kingdgrizzle@gmail.com>",
 		"SpaceEEC <spaceeec@yahoo.com>",
-		"Aura Román <kyradiscord@gmail.com>"
+		"Aura Román <kyradiscord@gmail.com>",
+		"Vinayak Kulkarni <inbox.vinayak@gmail.com>"
```

Comment:
this section is for the core maintainers of the project.

```suggestion
		"Aura Román <kyradiscord@gmail.com>"
```

Reply:
Cool, will remove it 😃

Reply:
- [x] Done in 199b242